remote/remote_test.go

```diff
@@ -196,6 +196,31 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 		})
 	})
 
+	when("#Labels", func() {
+		when("image exists", func() {
```

Comment:
Same comment as above regarding cases where the image does not exist or no labels have been set.

Reply:
See previous response, I don't think we are in danger of a pointer exception here. `nil` behaves like an empty map already.